### PR TITLE
forgot password form

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
Added forgot password form

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -39,7 +40,7 @@ function App() {
     routes = (
       <Switch>
         <Route path="/" exact>
-          <Users />
+          <ForgotPassword></ForgotPassword>
```
Comment (akhil-naidu) [inline client/src/App.js:46]: Why the Users component is replaced with ForgotPassword Component?

Also, why the Home page redirecting component is named as Forgot password, even if has all the logic.

**Review by akhil-naidu — COMMENTED** [review]
Any gotcha?

**Comment by ajitsinghkaler** [comment]
I was waiting for you to merge my custom components brach

It already contain my page

**Comment by akhil-naidu** [comment]
> I was waiting for you to merge my custom components brach
> 
> It already contain my page

Shall I merge this, are you going to replace Users back in the root?

**Comment by ajitsinghkaler** [comment]
No I was just waiting for you to merge my custom components branch

**Pull request merged**